common/types/gql-types.ts

```diff
@@ -1866,7 +1866,7 @@ export type QueryReservationsArgs = {
   offset?: InputMaybe<Scalars["Int"]>;
   onlyWithPermission?: InputMaybe<Scalars["Boolean"]>;
   orderBy?: InputMaybe<Scalars["String"]>;
-  paymentStatus?: InputMaybe<Array<InputMaybe<Scalars["String"]>>>;
+  orderStatus?: InputMaybe<Array<InputMaybe<Scalars["String"]>>>;
```

Comment:
wasn't this already fixed O_O

Reply:
Suppose they're still at PR level. `orderStatus` is the current one, right?

Reply:
Yep. strange. the pr has gone through like a week ago.